docs/source/algorithms/Fit-v1.rst

```diff
@@ -178,6 +178,37 @@ Fitting to data in a MatrixWorkspace
 The error values in the input workspace are used to weight the data in
 the fit. Zero error values are not allowed and are replaced with ones.
 
+Fitting to histogram data
+#########################
+
+If the input MatrixWorkspace contains histogram data then there are two
+way to evalue the fit function on it and it can be set via the `EvaluationType`
+property. If `EvaluationType` is "CentrePoint" (which is the default) the
+function is evaluated at the centre of the bin. If `EvaluationType` is set to
+"Histogram" then the function is integrated within each bin and these integrals
+are used to evaluate the cost function.
+
+The "Histogram" evaluation type can give more accurate results when fitting
+histograms with very large bins. It also less sensitive to a particular binning.
```

Comment:
Nice figure! 👍 Do you maybe want to point out in more detail that in particular the estimate for FWHM has improved a lot?

(I am mostly traveling this week and have some other appointments, if no one picks this up I could probably review next week).